src/CodeFormatter/App.config

```diff
@@ -37,6 +37,34 @@
         <assemblyIdentity name="Microsoft.CodeAnalysis.CSharp.Workspaces" publicKeyToken="31bf3856ad364e35" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-0.7.0.0" newVersion="0.7.0.0" />
       </dependentAssembly>
+      <dependentAssembly>  
```

Comment:
These binding redirects are needed to analyze some projects which have build tasks built against MSBuild 4.0. Since we will run with MSBuild 14.0, without these redirects those tasks would fail and we'll end up with a broken workspace otherwise.